### PR TITLE
Inline stable grapheme fastpaths

### PR DESCRIPTION
**Pull request by dabrahams**
/cc @milseman

### CODE REVIEW

**Comment by dabrahams** [comment]
@swift-ci Please benchmark

**Comment by dabrahams** [comment]
@swift-ci Please smoke test

**Comment by dabrahams** [comment]
Not sure what I got wrong here but the benchmarks should still be instructive

**Comment by dabrahams** [comment]
@swift-ci please test

**Comment by dabrahams** [comment]
@swift-ci please test

**Comment by dabrahams** [comment]
@swift-ci Please benchmark

**Comment by dabrahams** [comment]
@swift-ci Please benchmark

**Comment by dabrahams** [comment]
We could totally think about shifting the boundary to just fastPath ASCII, but this looks like an improvement already.

File: stdlib/public/core/StringCharacterView.swift [hunk 1]
```diff
@@ -405,7 +425,7 @@ extension String.CharacterView : BidirectionalCollection {
     return legacyGraphemeForward(
       start: start, end: end, startIndexUTF16: startIndexUTF16
     )
-  }
+	}
```
Comment (moiseev) [inline stdlib/public/core/StringCharacterView.swift]: This looks wrong. Is this a TAB?

Comment (dabrahams) [inline stdlib/public/core/StringCharacterView.swift]: It appears so.

**Pull request merged**